parsing/location.mli

```diff
@@ -203,9 +211,9 @@ val default_warning_reporter: t -> Warnings.t -> report option
 
 (** {2 Printing warnings} *)
 
-val formatter_for_warnings : formatter ref
+val log_for_warnings : Misc.Log.t ref
```

Comment:
same comment here are with `report_printer`, unclear we gain by allowing people to hook the JSON reporter, at the cost of changing the type here.